net/interfaces/interfaces.go

```diff
@@ -402,13 +426,23 @@ func prefixesEqual(a, b []netaddr.IPPrefix) bool {
 	return true
 }
 
-// FilterInteresting reports whether i is an interesting non-Tailscale interface.
-func FilterInteresting(i Interface, ips []netaddr.IPPrefix) bool {
+// UseInterestingInterfaces reports whether i is an interesting non-Tailscale interface.
+func UseInterestingInterfaces(i Interface, ips []netaddr.IPPrefix) bool {
 	return !isTailscaleInterface(i.Name, ips) && anyInterestingIP(ips)
 }
 
-// FilterAll always returns true, to use EqualFiltered against all interfaces.
-func FilterAll(i Interface, ips []netaddr.IPPrefix) bool { return true }
+// UseInterestingIPs reports whether ip is an interesting IP address.
```

Comment:
This name sounds really weird without knowing about type IPFilter.

Maybe it should be:

    // UseInterestingIPs returns an IPFilter that only matches ....
    func UseInterestingIPs() IPFilter {
         return ...
    }

But also: if this is now exported API, it's all the more important to document what an "interesting" IP means.